documentation/API_documentation/QoD_API.md

```diff
@@ -0,0 +1,178 @@
+<br>
+
+#### QoD Delete QoS Session
+
+| **Deleting QoS session** |
+| ---------------------------- |
+| **HTTP Request**<br>  DELETE\<base-url>/qod-api/v0/sessions/{sessionId}<br>**Query Parameters**<br>  No query parameters are defined.<br>**Path Parameters**<br>  sessionId: Session ID that needs to be terminated.<br>**Request Body Parameters**<br>  No request body parameters are defined.<br><br>**Response**<br> **204:** Session deleted<br> **401:** Un-authorized. <br> **403:** Forbidden.<br> **404:** Session not found.<br> **503:** Service temporarily unavailable.|
+
+### 4.4 Errors
+
+Since CAMARA QoD API is based on REST design principles and blueprints, well defined HTTP status codes and families specified by community are followed [[4]](#4).
+
+Details of HTTP based error/exception codes for the QoD API are described in Section 4.3 of each API REST based method.
+Following table provides an overview of common error names, codes, and messages applicable to QoD API.
+
+| No | Error Name | Error Code | Error Message |
+| --- | ---------- | ---------- | ------------- |
+|1	|400 |	INVALID_INPUT |	"Expected property is missing: ueId.msisdn" |
+|2	|400 |	INVALID_INPUT |	"Expected property is missing: ueId.ipv4addr" |
+|3	|400 |	INVALID_INPUT |	"Expected property is missing: ueId.ipv4addr or ueId.ipv6addr" |
+|4	|400 |	INVALID_INPUT |	"Expected property is missing: uePorts" |
+|5	|400 |	INVALID_INPUT |	"Expected property is missing: qos" |
+|6	|400 |	INVALID_INPUT |	"Ranges not allowed: uePorts" |
+|7	|401 |	UNAUTHORIZED |	"No authorization to invoke operation" |
+|8	|403 |	FORBIDDEN |	"Operation not allowed" |
+|9	|404 |	NOT_FOUND |	"Session Id does not exist" |
+|10	|409 |	CONFLICT |	"Another session is created for the same UE" |
```

Comment:
Question: I thought that an API developer can create multiple QoD sessions with different Flow Descriptions (e.g. different ports) to the same device.

Reply:
@tlohmar Thanks for excellent observation ... need to be corrected with an issue and PR in the YAML first. What was actually meant (most probably) "There is an overlapping session existing for the same UE" ... but tbd.